calicoctl-wait/calicoctl-wait.go

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2020 Tigera, Inc. All rights reserved.
```

Comment:
What is this for?

Reply:
The calico-wait tool is used to replace Linux utils used in the container image and go for a pure scratch image.